devtools/dev-requirements_conda.yml

```diff
@@ -0,0 +1,45 @@
+name: desc-env
```

Comment:
When we add new requirements, do we have to add them to both `devtools/dev-requirements.txt` and `devtools/dev-requirements_conda.yml`?

Reply:
Yes. One is for if you do `pip install -r requirements.txt`. The other is when you use conda environments instead. I could write a shell script to sync changes between them in `devtools/`. That wouldn't make it in this pr.

I think the conda version is nice because it sets up the environment for you. It's worse to manually create a conda environment then do `conda install` and `pip install ` because of package dependency issues. I figured if people aren't installing desc with `pip install desc-opt`, they are probably using conda anyway.